components/Books.js

```diff
@@ -60,15 +39,14 @@ class Books extends React.Component {
         url =
           "The service will fetch the original issue from the article ID (not just the article)";
         break;
+
+      default:
+        break;
     }
     return url;
   };
 
-  /**
-   * Makes a request from Google Books API based on the entered book Id
-   * If the book Id is not valid the request resolves with an error message
-   */
-  validateGoogleBook = (enteredId) => {
+  const validateGoogleBook = (enteredId) => {
```

Comment:
Looks like this function is unused and can be removed

Reply:
I have removed the unused functions.